featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -789,34 +788,18 @@ def gen_feat_names(self):
         for base_feature in feature.base_features:
             assert base_feature.unique_name() != join_time_split.unique_name()
```

Comment:
I think we can delete this for loop checking that we don't stack on top of multi-output features, combine the dfs call above with the new one written below,  and use the new for loop checking that we do stack on multi-output features